R/dash.R

```diff
@@ -498,11 +508,55 @@ Dash <- R6::R6Class(
         TRUE
       })
 
+      dash_reload_hash <- paste0(self$config$routes_pathname_prefix, "_reload-hash")
+      route$add_handler("get", dash_reload_hash, function(request, response, keys, ...) {
+        modified_files <- private$modified_since_reload
+        hard <- TRUE
+        
+        if (is.null(modified_files)) {
+          # dash-renderer requires that this element not be NULL
+          modified_files <- list()
+        }
+        
+        resp <- list(files = modified_files,
+                     hard = hard,
+                     packages = c("dash_renderer", 
+                                  unique(
+                                    vapply(
+                                      private$dependencies, 
+                                      function(x) x[["name"]], 
+                                      FUN.VALUE=character(1), 
+                                      USE.NAMES = FALSE)
+                                  )
+                     ),
+                     reloadHash = self$config$reload_hash)
+        response$body <- to_JSON(resp)
+        response$status <- 200L
+        response$type <- 'json'
+        # reset the field for the next reloading operation
+        private$modified_since_reload <- list()
+        TRUE
+      })
+      
       router$add_route(route, "dashR-endpoints")
       server$attach(router)
-
+      
```

Comment:
🙄

Reply:
fixed in aa1ad07c4e5154ee96688b0158c3a1b22d9c7e0b